packages/validator/src/capability2.js

```diff
@@ -0,0 +1,109 @@
+import * as API from '@ucanto/interface'
```

Comment:
Is this `capability2.js` file imported anywhere?

Reply:
what is intended difference between `capability.js` and `capability2.js` (if any)

Reply:
oops, I did not intend to include that file